Modules/main.c

```diff
@@ -1441,6 +1442,8 @@ _PyMainInterpreterConfig_Copy(_PyMainInterpreterConfig *config,
 {
     _PyMainInterpreterConfig_Clear(config);
 
+    config->install_signal_handlers = config2->install_signal_handlers;
```

Comment:
I would prefer to use a COPY_ATTR() macro here:

#define COPY_ATTR(ATTR) config->ATTR = config2->ATTR

Rename the existing macro to COPY_OBJ_ATTR(): see _PyCoreConfig_Copy().